File: fe2-android/app/src/test/java/com/github/dedis/popstellar/model/objects/ServerTest.java
package com.github.dedis.popstellar.model.objects;

import static org.junit.Assert.assertEquals;

import com.github.dedis.popstellar.model.objects.security.PublicKey;
import org.junit.Test;

public class ServerTest {

  public static final PublicKey RANDOM_KEY =
      new PublicKey("oOcKZjUeandJOFVgn-E6e-7QksviBBbHUPicdzUgIm8");
  public static final String RANDOM_ADDRESS = "ws://10.0.2.2:9000/organizer/client";

  public static final Server SERVER = new Server(RANDOM_ADDRESS, RANDOM_KEY);

  @Test
  public void getPublicKeyTest() {
    assertEquals(RANDOM_KEY, SERVER.getPublicKey());
  }

  @Test
  public void getServerAddressTest() {
    assertEquals(RANDOM_ADDRESS, SERVER.getServerAddress());
  }
}

File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/objects/ServerTest.kt
package com.github.dedis.popstellar.model.objects

import com.github.dedis.popstellar.model.objects.security.PublicKey
import org.junit.Assert
import org.junit.Test

class ServerTest {
  @Test
  fun publicKeyTest() {
    Assert.assertEquals(RANDOM_KEY, SERVER.publicKey)
  }

  @Test
  fun serverAddressTest() {
    Assert.assertEquals(RANDOM_ADDRESS, SERVER.serverAddress)
  }

  companion object {
    val RANDOM_KEY = PublicKey("oOcKZjUeandJOFVgn-E6e-7QksviBBbHUPicdzUgIm8")
    const val RANDOM_ADDRESS = "ws://10.0.2.2:9000/organizer/client"
    val SERVER = Server(RANDOM_ADDRESS, RANDOM_KEY)
  }
}
